### PR TITLE
drush options and missing target_bundle fallback

### PR DESCRIPTION
**Pull request by irmela**

### CODE REVIEW

File: entityrelationships.drush.inc [hunk 1]
```diff
@@ -7,17 +7,24 @@ function entityrelationships_drush_command() {
   $items['entityrelations'] = array(
     'description' => 'Generate a graph of the entities and fields.',
     'aliases' => array('ers'),
+    'options' => array(
+      'include_fields' => 'Include entity fields',
```
Comment (amitaibu) [inline entityrelationships.drush.inc]: maybe just `Include fields` instead?

File: entityrelationships.drush.inc [hunk 2]
```diff
@@ -7,17 +7,24 @@ function entityrelationships_drush_command() {
...
+      'entity_type' => 'Include single entity type',
```
Comment (amitaibu) [inline entityrelationships.drush.inc]: Maybe `Select a single entity type`

File: entityrelationships.drush.inc [hunk 3]
```diff
@@ -7,17 +7,24 @@ function entityrelationships_drush_command() {
...
       'drush entityrelations | dot -Gratio=0.7 -Eminlen=2 -T png -o ./test.png' => 'Generate the Entity-Relationship graph for the current site and export it has a PNG image.',
+      'drush entityrelations --include_fields --entity_type=node | dot -Gratio=0.7 -Eminlen=2 -T png -o ./test.png' => 'Generate the Entity-Relationship graph for a entity type site including fields and export it has a PNG image.',
     ),
   );
   return $items;
 }
 
 function drush_entityrelationships_entityrelations() {
-  $graph =  entityrelationships_entitygraph(TRUE);
+  $include_fields = drush_get_option('include_fields');
```
Comment (amitaibu) [inline entityrelationships.drush.inc]: I think mostly for readability we should `drush_get_option('include_fields', FALSE);` - so it's clear what the default value is.

**Comment by amitaibu** [comment]
Thanks -- added a few comments. Happy to see others use this :)

**Comment by irmela** [comment]
Thanks a lot for your comments. I just implemented them.

**Comment by amitaibu** [comment]
Merged, thanks.

@bricel can you push `master` to your d.o repo as-well

**Pull request merged**